src/components/course/data/tests/utils.test.jsx

```diff
@@ -386,31 +393,20 @@ describe('getMissingSubsidyReasonActions', () => {
       enterpriseAdminUsers: [{ email: 'admin@example.com' }],
       contactEmail: undefined,
     });
-    render(ActionsComponent);
+    renderWithIntl(ActionsComponent);
     const ctaBtn = screen.getByText('Contact administrator');
     expect(ctaBtn).toBeInTheDocument();
     expect(ctaBtn.getAttribute('href')).toEqual('mailto:admin@example.com');
   });
 
-  it.each([
-    DISABLED_ENROLL_REASON_TYPES.NO_SUBSIDY,
-    DISABLED_ENROLL_REASON_TYPES.POLICY_NOT_ACTIVE,
-    DISABLED_ENROLL_REASON_TYPES.LEARNER_NOT_IN_ENTERPRISE,
-    DISABLED_ENROLL_REASON_TYPES.CONTENT_NOT_IN_CATALOG,
-    DISABLED_ENROLL_REASON_TYPES.NOT_ENOUGH_VALUE_IN_SUBSIDY,
-    DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_EXPIRED,
-    DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_SEATS_EXHAUSTED,
-    DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_LICENSE_NOT_ASSIGNED,
```

Comment:
[clarification] It looks like all but `DISABLED_ENROLL_REASON_TYPES.SUBSCRIPTION_LICENSE_NOT_ASSIGNED` were removed from the test. I'm wondering if this necessary? For example, by doing so, are we losing test coverage that `getMissingSubsidyReasonActions` behaves correctly for all the other disabled enroll reason types?